src/main/java/org/tools4j/hoverraft/message/AppendResponse.java

```diff
@@ -37,6 +37,14 @@
 
     AppendResponse successful(boolean successful);
 
+    AppendResponse serverId(int serverId);
+
+    int serverId();
+
+    AppendResponse matchLogIndex(long matchLogIndex);
+
+    long matchLogIndex();
```

Comment:
should we not better name this one ``appendedLogIndex()`` --- or maybe make it a LogKey and call it ``appendedLogKey()`` ?